src/param/param_slash.c

```diff
@@ -75,6 +79,45 @@ static void param_completer(struct slash *slash, char * token) {
 		skip_prefix = "cmd add";
 	}
 
+	if (skip_prefix == NULL) {
+		fprintf(stderr, "%s() does not support the line/command: '%s'", __func__, slash->buffer);
+		assert(false);  // Assert because this should be a compile-time error on part of the programmer, not the user.
+		return;
+	}
+
+	{   /* Keep optparse parser in local scope, to prevent use-after-free */
+
+		#define SLASH_ARG_MAX		16	/* Maximum number of arguments */
+		char *argv[SLASH_ARG_MAX];
+		int argc = 0;
+		char *args = strdup(slash->buffer + strlen(skip_prefix));
```

Comment:
I don't see a corresponding call to "free", am I missing something?

Reply:
Nope. It was I, who was missing something :)
I must've simply gotten too used to __attribute__((cleanup())) for my own good.
I've added a corresponding free() in the newest commit (8962b67)